testing/sssd/APKBUILD

```diff
@@ -0,0 +1,105 @@
+# Contributor: Vince Mele <vmele@inoc.com>
+# Maintainer: Vince Mele <vmele@inoc.com>
+pkgname=sssd
+pkgver=1.15.3
+pkgrel=0
+pkgdesc="System Security Services Daemon"
+url="https://pagure.io/SSSD/sssd"
+arch="all"
+license="GPL3"
+depends="bind \
+	c-ares \
+	curl \
+	cyrus-sasl-gssapi \
+	ding-libs \
+	http-parser \
+	jansson \
+	libnl \
+	libunistring \
+	nss \
+	"
```

Comment:
I am not sure how many of those runtime depends you need to specify. `abuild` will autodetect shared libraries for you, but if you for example need `/usr/bin/curl` you need to specify `curl` in `depends`.

Reply:
@ncopa it would seem arch-linux also has curl as a runtime dependency, that dependency was added in arch for 1.15.3 
https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/sssd

it would seem the rest of the depends list quite closely matches that of the arch packaging.